src/defaults.js

```diff
@@ -2,7 +2,7 @@
 const hasDOM = typeof document !== "undefined";
 
 export default {
-	gamut_mapping: "lch.c",
```

Comment:
This changes the default `gamut_mapping`, which is a breaking change. Do we instead want to phase this in?

Reply:
Removing it as an option would be a much more breaking change. But I don't know how much the arbitrary channel stuff is being used.

Reply:
I think it's fine to change the default, but we should keep the channel method, at least for now.